### PR TITLE
Add docs.

### PR DESCRIPTION
**Pull request by dewniMW**

### CODE REVIEW

File: en/docs/guides/access-delegation/authorization-code-with-pkce.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# Authorization Code Grant With PKCE
+
+PKCE is a recommended security measure used to mitigate a [code interception attack](../../../deploy/mitigate-attacks/mitigate-authorization-code-interception-attacks/). 
+This page guides you through using [Authorization Code Grant](../../../../concepts/authorization/authorization-code-grant/) with PKCE
```
Comment (gomathyk) [inline en/docs/guides/access-delegation/authorization-code-with-pkce.md:4]:
```suggestion
This page guides you through using [Authorization Code Grant](../../../concepts/authorization/authorization-code-grant/) with PKCE 
```

File: en/docs/guides/access-delegation/client-credentials.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+```
+
+You will receive the following response with the access token.
+
+```
+{"access_token":"16ab408c-0f31-3321-8bed-313e836df373","token_type":"Bearer","expires_in":2986}
+```
+
+!!! info "Related Topics"
+    - [Concept: Client Credentials Grant](../../../../concepts/authorization/client-credential-grant/)
```
Comment (gomathyk) [inline en/docs/guides/access-delegation/client-credentials.md:54]:
```suggestion
    - [Concept: Client Credentials Grant](../../../concepts/authorization/client-credential-grant/)
```

File: en/docs/guides/access-delegation/client-credentials.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    - [Guide: Advanced OpenID Connect Configurations](../../login/oauth-app-config-advanced)
```
Comment (gomathyk) [inline en/docs/guides/access-delegation/client-credentials.md:55]:
```suggestion
    - [Guide: Advanced OpenID Connect Configurations](../../../guides/login/oauth-app-config-advanced)
```

File: en/docs/guides/access-delegation/implicit.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+You will receive the ID token. 
+
+```
+http://wso2is.local:8080/playground2/oauth2client#access_token=317c19b3-73e3-3906-8627-d1c952856b5d&token_type=Bearer&expires_in=3600
+```
+
+!!! info "Related Topics"
+    - [Concept: Implicit Grant](../../../../concepts/authorization/implicit-grant/)
+    - [Guide: Advanced OpenID Connect Configurations](../../login/oauth-app-config-advanced)
```
Comment (gomathyk) [inline en/docs/guides/access-delegation/implicit.md:60]:
```suggestion
    - [Guide: Advanced OpenID Connect Configurations](../../../guides/login/oauth-app-config-advanced)
```

File: en/docs/guides/access-delegation/oidc-basic-client-profile.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
+# OpenID Connect Basic Client Profile
+
+This section provides information about the expected requests and the
+relevant responses that the WSO2 Identity Server would generate for the
+[OpenID Connect Basic Client](../../../concepts/authentication/basic-client-profile/) flow.
+ 
+
+### Register a service provider
```
Comment (gomathyk) [inline en/docs/guides/access-delegation/oidc-basic-client-profile.md:8]: Please make the headers h2

File: en/docs/guides/access-delegation/oidc-basic-client-profile.md [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+            "preferred_username":"admin",
+            "given_name":"admin",
+            "profile":"https://wso2.com",
+            "country":"Sri Lanka"
+        }
+        ``` 
+
+!!! info "Related Topics"
+    - [Concept: OpenID Connect Implicit Client](../../../concepts/authentication/basic-client-profile/)
+    - [Guide: Advanced OpenID Connect Configurations](../../login/oauth-app-config-advanced)
```
Comment (gomathyk) [inline en/docs/guides/access-delegation/oidc-basic-client-profile.md:114]:
```suggestion
    - [Guide: Advanced OpenID Connect Configurations](../../../guides/login/oauth-app-config-advanced)
```

File: en/docs/guides/access-delegation/oidc-implicit-client-profile.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+```
+
+!!! info "Related Topics"
+    - [Concept: OpenID Connect Implicit Client](../../../concepts/authentication/implicit-client-profile/)
+    - [Guide: Advanced OpenID Connect Configurations](../../login/oauth-app-config-advanced)
```
Comment (gomathyk) [inline en/docs/guides/access-delegation/oidc-implicit-client-profile.md:86]:
```suggestion
    - [Guide: Advanced OpenID Connect Configurations](../../../guides/login/oauth-app-config-advanced)
```

File: en/docs/guides/access-delegation/password-grant.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+
+You will receive the following response with the access token and refresh token. 
+
+```
+{"access_token":"16ab408c-0f31-3321-8bed-313e836df373","refresh_token":"3c285b4f-ec29-3751-9ced-74c92061b327","token_type":"Bearer","expires_in":3600}
+```
+
+!!! info "Related Topics"
+    - [Concept: Resource Owner Password Credentials Grant](../../../../concepts/authorization/resource-owner-grant/)
+    - [Guide: Advanced OpenID Connect Configurations](../../login/oauth-app-config-advanced)
```
Comment (gomathyk) [inline en/docs/guides/access-delegation/password-grant.md:55]:
```suggestion
    - [Guide: Advanced OpenID Connect Configurations](../../../guides/login/oauth-app-config-advanced)
```

File: en/mkdocs.yml [hunk 9]
```diff
@@ -256,8 +256,15 @@ nav:
...
-          - 'Web Applications': guides/access-delegation/overview.md
-          - 'Client Side Applications': guides/access-delegation/oauth-grant-types.md
+          - 'Web Applications':
+            - 'Authorization Code' : guides/access-delegation/authorization-code.md
+            - 'Client Credentials': guides/access-delegation/client-credentials.md
+            - 'Resource Owner Password Credentials' : guides/access-delegation/password-grant.md
+            - 'SAML2 Bearer Assertion': guides/access-delegation/saml2-bearer-assertion-profile.md
+          - 'Client Side Applications':
+            - 'Implicit': guides/access-delegation/implicit.md
+            - 'Authorization Code ': guides/access-delegation/authorization-code-without-pkce.md
```
Comment (gomathyk) [inline en/mkdocs.yml:266]:
```suggestion
            - 'Authorization Code': guides/access-delegation/authorization-code-without-pkce.md
```

File: en/mkdocs.yml [hunk 10]
```diff
@@ -256,8 +256,15 @@ nav:
         - 'Interactive Account Linking': guides/account-linking/interactive-account-link.md
       - 'Access Delegation':
         - 'OAuth 2.0':
-          - 'Web Applications': guides/access-delegation/overview.md
-          - 'Client Side Applications': guides/access-delegation/oauth-grant-types.md
+          - 'Web Applications':
+            - 'Authorization Code' : guides/access-delegation/authorization-code.md
+            - 'Client Credentials': guides/access-delegation/client-credentials.md
+            - 'Resource Owner Password Credentials' : guides/access-delegation/password-grant.md
```
Comment (gomathyk) [inline en/mkdocs.yml:262]:
```suggestion
            - 'Resource Owner Password Credentials': guides/access-delegation/password-grant.md
```

File: en/mkdocs.yml [hunk 11]
```diff
@@ -256,8 +256,15 @@ nav:
         - 'Interactive Account Linking': guides/account-linking/interactive-account-link.md
       - 'Access Delegation':
         - 'OAuth 2.0':
-          - 'Web Applications': guides/access-delegation/overview.md
-          - 'Client Side Applications': guides/access-delegation/oauth-grant-types.md
+          - 'Web Applications':
+            - 'Authorization Code' : guides/access-delegation/authorization-code.md
```
Comment (gomathyk) [inline en/mkdocs.yml:260]:
```suggestion
            - 'Authorization Code': guides/access-delegation/authorization-code.md
```

**Review by gomathyk — CHANGES_REQUESTED** [review]

File: en/docs/guides/access-delegation/authorization-code-with-pkce.md (hunk 1 above)
Comment (dewniMW) [reply en/docs/guides/access-delegation/authorization-code-with-pkce.md:4]: isn't the suggested path wrong?

File: en/docs/guides/access-delegation/client-credentials.md (hunk 2 above)
Comment (dewniMW) [reply en/docs/guides/access-delegation/client-credentials.md:54]: the suggested path is incorrect

**Review by gomathyk — APPROVED** [review]

**Pull request merged**